### PR TITLE
ISSD-1804 [Marketplace] Order Details Tab Content

### PR DESCRIPTION
**Pull request by claricebrasil**

### CODE REVIEW

**Comment by claricebrasil** [comment]
ci:test:sf

**Comment by claricebrasil** [comment]
ci:test:sf

**Comment by claricebrasil** [comment]
ci:test:sf

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/App/App.tsx [hunk 1]
```diff
@@ -3,29 +3,176 @@
...
+		(field: {name: string}) => field.name === 'Project Name'
+	);
+
+	const formatSKUName = (sku: string) => {
+		const skuLowerCased = sku.toLowerCase();
 
-	const {data} = useGetProductByOrderId(orderId);
-	const product = data?.product;
+		return skuLowerCased.charAt(0).toUpperCase() + skuLowerCased.slice(1);
+	};
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/App/App.tsx:29]: @claricebrasil this is used for Capitalize the text?

We could use `text-capitalize` in CSS, it's better

**Review by kevenleone — CHANGES_REQUESTED** [review]
Good job @claricebrasil just a few adjustment is needed

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/App/App.tsx (hunk 1 above)
Comment (claricebrasil) [reply workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/App/App.tsx]: Oh, good point. Thank you, Keven!

**Comment by claricebrasil** [comment]
ci:test:sf

**Comment by claricebrasil** [comment]
ci:test:stop

**Comment by claricebrasil** [comment]
ci:stop

**Comment by claricebrasil** [comment]
resent: #5202

**Pull request closed without merging**